### DISCUSSION TITLE
Full OTA vs Incremental OTA - risks

### DISCUSSION THREAD

**Discussion by PJanisio** [quintenvandamme/nothing_archive#1] [Q&A] [answered]
Hi!

Currently for 2a phone last firmware (2.5.5) is during stage implementation. Some of regions are waiting several days and still have no update.

**Question:** What is a risk to download and install current Incremental OTA: `2.5.5	Pacman-U2.5-240410-1238`
Is there any? Which issues can be expected?

Thanks for answer!

**Comment by quintenvandamme** [comment, marked as answer]
Hi!

The Incremental update from 2.5.4A to 2.5.5 should be safe to install.

The difference between an incremental and full ota is that the incremental only contains the updated files from a previous version and a full ota is the entire build. So full ota's can be installed on any version of Nothing OS.

These builds are also downloaded from the official google ota api urls and archived on archive.org and the sha1 hash of the file should be always the same as the zip file name, If not the ota file has been tampered with.

**Reply by PJanisio** [reply]
Thanks for detailed reply. Just thinking than why there is staging update for some of regions/devices which has already 2.5.4a installed.

**Comment by PJanisio** [comment]
Closing.
